PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Assignee Section
+
+## Description
+[FILL]
+
+Closes Issue : [#FILL]
+
+Approval from at least one software team lead is necessary before merging.
+
+# Reviewer Section
+
+Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.
```

Comment:
I would also add something like: `For PRs that don't actually change code/functionality running these tests are not necessary`

Reply:
done